### DISCUSSION TITLE
Sorting Collections

### DISCUSSION THREAD

**Discussion by TomHeckl** [vuejs/vuefire#1267] [Questions and Help] [answered]
Hi there,
I´m already new to Vuefire V3 so I have a very simple question.
I want to retrieve a list of participants from a firebase collection and use it with vuefire.
So it is easy to fetch the data, but I can´t find an easy way to sort the data by a special field like name or birthday.
So my code looks something like that:
`const firebaseApp = initializeApp(firebaseConfig);`
`const auth = getAuth(firebaseApp);`
`const db = getFirestore();`
`const participants_ref= collection(db, 'participants')`
`const participants= useCollection (participants_ref)`
`export { auth, participants }`
This works fine. 
I´m looking for somathing like

`const Fahrschueler_Ref = collection(db, 'Fahrschueler')`**.orderBy('name')**

Would be great if someone could give me a hint.

Thanks
Tom

**Comment by TomHeckl** [comment]
Found an answer.
Thx

**Reply by posva** [reply]
It would be helpful for others if you post the actual answer 😄

**Comment by TomHeckl** [comment, marked as answer]
The answer was so simple, I thought, surely anyone can solve this easily right away. I simply used a query...
```ts
import { query, collection, orderBy } from 'firebase/firestore'
import { useCollection from 'vuefire'

const q = query(collection(db, 'participant'), orderBy('name'))
const FS_sorted = useCollection(q)
```
